constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/motion/widget/MotionScene.java

```diff
@@ -648,7 +648,7 @@ public boolean isEnabled() {
          *
          * @param enable
          */
-        public void setEnable(boolean enable) {
+        public void setEnabled(boolean enable) {
```

Comment:
This is part of the existing public API in stable versions, definitely need to keep the existing method and add deprecation notice to make this change.

Reply:
Ok, I put it back with deprecation.